sound/soc/sof/pm.c

```diff
@@ -435,12 +435,66 @@ EXPORT_SYMBOL(snd_sof_set_d0_substate);
 
 int snd_sof_resume(struct device *dev)
 {
+	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
```

Comment:
fix commit message

ASoC: SOF: PM: Add support for DSP D0i3 state when entering S0ix

When system is entering into S0ix, the PCI device may transition to the D0i3 substate instead of
D3. In D0i3, some always-on functionality can be enabled, such as acoustic event detection, voice activity detection or hotwording. When an event is detected, the DSP firmware can wake-up the device for a transition to D0 with an interrupt.

Reply:
@keyonjie you have not address the commit message update from Pierre.

Reply:
Thanks, done in the new version.